src/pomelo_fmt_plug.c

```diff
@@ -3,6 +3,9 @@
  * contest by Dhiru Kholia.
  */
 
+// Enable this format only on little-endian systems
+#ifdef ARCH_LITTLE_ENDIAN
```

Comment:
`#if ARCH_LITTLE_ENDIAN`

We're using these `arch.h` macros as 0 vs. 1, not as defined vs. undefined.

Reply:
It seems that I did a sloppy job with this PR ;(